tests/libres_tests/res/enkf/test_enkf_load_results_manually.py

```diff
@@ -70,3 +73,26 @@ def test_load_results_from_run_context(self):
             self.assertEqual(24, loaded)
             self.assertEqual(25, len(expected))
             self.assertEqual(25, len(realisations))
+
+
+class TestLoadResultsManuallyLogtest:
+    """
+    This little test does not depend on Equinor-data and only verifies
+    the lazy_load flag in forward_load_context plus memory-logging
+    """
+
+    def test_load_results_manually(self, setup_case, caplog, monkeypatch):
+        for val in ["false", "true"]:
+            res_config = setup_case("local/snake_oil", "snake_oil.ert")
+            ert = EnKFMain(res_config)
+            load_from = ert.getEnkfFsManager().getFileSystem("default_0")
+            ert.getEnkfFsManager().switchFileSystem(load_from)
+            realisations = BoolVector(default_value=False, initial_size=25)
+            realisations[0] = True  # only need one to test what we want
+            with caplog.at_level(logging.INFO):
+                loaded = ert.loadFromForwardModel(realisations, 0, load_from)
```

Comment:
This only appears to be used in `load_results_panel.py` in `ert`?

Reply:
Ref our conversation: Should we leave the test as is?